tools/tpm2_quote.c

```diff
@@ -158,7 +158,8 @@ static tool_rc process_output(ESYS_CONTEXT *ectx) {
         }
 
         // Gather PCR values from the TPM (the quote doesn't have them!)
-        rc = pcr_read_pcr_values(ectx, &ctx.pcr_selections, &ctx.pcrs);
+        rc = pcr_read_pcr_values(ectx, &ctx.pcr_selections, &ctx.pcrs,
+            &ctx.cp_hash, ctx.parameter_hash_algorithm);
```

Comment:
```suggestion
            NULL, TPM2_ALG_ERROR);
```

Reply:
A call to pcr_read with the same variable name will clobber cpHash for tpm2_quote.